etc/ci/performance/submit_to_perfherder.py

```diff
@@ -317,8 +317,7 @@ def submit(perf_data, failures, revision, summary, engine):
         'secret': os.environ['TREEHERDER_CLIENT_SECRET']
     }
 
-    client = TreeherderClient(protocol='https',
-                              host='treeherder.allizom.org',
+    client = TreeherderClient(server_url='https://treeherder.allizom.org',
```

Comment:
Sorry, we changed to `https://treeherder.mozilla.org`, can you help us change that.